ui/src/components/yaml/styles.module.less

```diff
@@ -324,17 +324,14 @@
       }
     }
 
-    .yaml_content_diagnosisContent {
+    .yaml_content_diagnosisBody {
+      position: relative;
       width: 400px;
-      box-sizing: border-box;
-      height: 100%;
       overflow-y: auto;
```

Comment:
Removed color, background, padding, and font-size. Ensure these changes don't break component appearance.